extra/exports

```diff
@@ -148,6 +148,10 @@ csv_feed
 csv_iterator_create
 csv_next
 csv_setopt
+datetime_now
+datetime_strftime
+datetime_to_string
+datetime_unpack
```

Comment:
Hmm, what about `swim_*`, or `say_*`, or numerous different prefixes, not `tnt_`. Sounds like you restrict it too much.

Reply:
These are also wrong and must be fixed. `say_*` functions seem to be a part of the public C API, see module.h. The same for tons of other public C methods. They can't be changed. Your new functions are not a part of the public C API, so please, hide them behind *tnt_* prefix. Or am I doing something wrong here? @Totktonada, that is the new policy now, right? We must use `tnt_` for all new non-public but exported symbols.

Reply:
The prefixing is about libraries that could be used in modules as well — to don't clash. I would also apply this rule for symbols exported just for usage with LuaJIT FFI from inside tarantool — when it is not a public API, but exported purely due to technical matters.

As for symbols provided by tarantool itself: I vote to have certain set of prefixes. Most of the public API functions are prefixed with `box_`. There are `fiber_` and `luaT_`. Not sure what is better to do about functions around tarantool specific objects. Maybe `box_`.

Before we'll discuss it futher, please, let me into the context: whether we export it for FFI (internal use) or as a real public API (which should be documented later and will be subject for backward compatibility)? If the latter was not asked to be a part of the task I suggest to postpone it.

Reply:
> The idea of exports (since recently) is that you should not expose functions not protected with a `tnt_` prefix. Otherwise they will conflict with some user's library.

I completely agree with the statement of this gentleman.

Reply:
Okay, if it is purely for internal use I would add a prefix. We use `tnt_` is such cases. Don't know, maybe it is not the best choice, but at least it'll be consistent with other functions exported only for internal use with LuaJIT FFI.